package.json

```
@@ -142,14 +142,14 @@
     "@graphql-codegen/typescript-operations": "4.0.1",
     "@graphql-eslint/eslint-plugin": "3.18.0",
     "@originjs/vite-plugin-commonjs": "1.0.3",
+    "@storybook/addon-actions": "7.4.0",
+    "@storybook/addon-essentials": "7.4.0",
+    "@storybook/addon-interactions": "7.4.0",
+    "@storybook/addon-links": "7.4.0",
+    "@storybook/addon-storyshots": "7.4.0",
+    "@storybook/react": "7.4.0",
+    "@storybook/react-vite": "7.4.0",
     "@sentry/vite-plugin": "2.6.2",
```

[nit] We should keep deps alphabetized
```suggestion
    "@sentry/vite-plugin": "2.6.2",
    "@storybook/addon-actions": "7.4.0",
    "@storybook/addon-essentials": "7.4.0",
    "@storybook/addon-interactions": "7.4.0",
    "@storybook/addon-links": "7.4.0",
    "@storybook/addon-storyshots": "7.4.0",
    "@storybook/react": "7.4.0",
    "@storybook/react-vite": "7.4.0",
```